### ISSUE TITLE
A cross-platform FindsByAttribute for accessibility id 

### ISSUE THREAD

**Issue by lanzelot1989** [appium/appium-dotnet-driver#176]
## Description

Is there a way to use a FindsByAttribute for both platforms and the Accessibility Id?
AppiumDriver.FindElementByAccessibilityId does this, but I cannot find an attribute for it to use in order to be initialized by PageFactory.InitElements(...) correctly.

I also tried to extend the FindsByUIAutomatorsAttribute myself which lead to a fallback to Id (not accessibility Id) in the code.

## Environment

* .NET client build version or git revision if you use some shapshot: 3.0.0.2
* Appium server version or git revision if you use some shapshot: 1.6.3
* Desktop OS/version used to run Appium if necessary: Windows 7
* Node.js version: 6.9.2
* Mobile platform/version under test: Android 7
* Real device or emulator/simulator: Emulator

## Code To Reproduce Issue [ Good To Have ]

Basically, I am looking for a cleaner version of
```
        private const string LoginButtonIdentifier = "btnLogin";
        [FindsByAndroidUIAutomator(Accessibility = LoginButtonIdentifier)]
        [FindsByIOSUIAutomation(Accessibility = LoginButtonIdentifier)]
        private IMobileElement<AppiumWebElement> LoginButton { get; set; }
```

**Comment by siggerzz** [comment]
Is your app a hybrid web app? If it is, I think this might work (assuming your driver is of type AppiumDriver<IWebElement>:

```
[FindsBy(How = How.Id, Using = "your-id-here")]
public IWebElement LoginButton { get; set; }
```

**Comment by lanzelot1989** [comment]
No, native apps (which were created with a cross-platform framework) and yes, the driver is of type AppiumDriver.

And yeah, that works. Although I also had to use
```
    // PageFactory.InitElements(Driver, this, new AppiumPageObjectMemberDecorator(new TimeOutDuration(new TimeSpan(0, 0, 0, 7, 0))));
    PageFactory.InitElements(Driver, this);
```
The specific decorator seems to trigger another part of the logic in PageFactory.
